gym-game2048/gym_game2048/envs/game_2048.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
-from numba import njit
+
 
 class Game2048:
-    def __init__(self, board_size: int, invalid_move_warmup=16, invalid_move_threshold=0.1, penalty=-512, dynamic_obstacle_interval=5):
+    def __init__(self, board_size: int, invalid_move_warmup=16, invalid_move_threshold=0.1, penalty=-512, dynamic_obstacle_interval=5, dynamic_obstacle_value=1):
```

Comment:
I made the dynamic_obstacle_value a variable that can be passed during initialization for better code readability. Let me know if you think this is overkill or if you'd prefer a different approach.